[2024-01-02T18:38:21.193+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: snow_dag.is_snow_api_ready manual__2024-01-02T18:38:14.858350+00:00 [queued]>
[2024-01-02T18:38:21.208+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: snow_dag.is_snow_api_ready manual__2024-01-02T18:38:14.858350+00:00 [queued]>
[2024-01-02T18:38:21.208+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 3
[2024-01-02T18:38:21.326+0000] {taskinstance.py:2192} INFO - Executing <Task(HttpSensor): is_snow_api_ready> on 2024-01-02 18:38:14.858350+00:00
[2024-01-02T18:38:21.332+0000] {standard_task_runner.py:60} INFO - Started process 111 to run task
[2024-01-02T18:38:21.334+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'snow_dag', 'is_snow_api_ready', 'manual__2024-01-02T18:38:14.858350+00:00', '--job-id', '47', '--raw', '--subdir', 'DAGS_FOLDER/servicenow_dag.py', '--cfg-path', '/tmp/tmpdnlsb3re']
[2024-01-02T18:38:21.338+0000] {standard_task_runner.py:88} INFO - Job 47: Subtask is_snow_api_ready
[2024-01-02T18:38:21.478+0000] {task_command.py:423} INFO - Running <TaskInstance: snow_dag.is_snow_api_ready manual__2024-01-02T18:38:14.858350+00:00 [running]> on host ba618e136646
[2024-01-02T18:38:21.773+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='prady1900' AIRFLOW_CTX_DAG_ID='snow_dag' AIRFLOW_CTX_TASK_ID='is_snow_api_ready' AIRFLOW_CTX_EXECUTION_DATE='2024-01-02T18:38:14.858350+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-02T18:38:14.858350+00:00'
[2024-01-02T18:38:21.774+0000] {http.py:130} INFO - Poking: api/now/table/incident
[2024-01-02T18:38:21.799+0000] {base.py:83} INFO - Using connection ID 'snow_api_id' for task execution.
[2024-01-02T18:38:24.482+0000] {base.py:293} INFO - Success criteria met. Exiting.
[2024-01-02T18:38:24.495+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=snow_dag, task_id=is_snow_api_ready, execution_date=20240102T183814, start_date=20240102T183821, end_date=20240102T183824
[2024-01-02T18:38:24.643+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-02T18:38:24.689+0000] {taskinstance.py:3281} INFO - 1 downstream tasks scheduled from follow-on schedule check
